metricbeat/metricbeat.full.yml

```diff
@@ -64,8 +64,8 @@ metricbeat.modules:
   # if true, exports the CPU usage in ticks, together with the percentage values
   #cpu_ticks: false
 
-  # EXPERIMENTAL: cgroups can be enabled for the process metricset.
-  #cgroups: false
+  # Enable collection of cgroup metrics from processes on Linux.
+  #process.cgroups.enabled: true
```

Comment:
I'd rather leave it out of the short config. I'd actually be in favor of removing more stuff from the short config. I feel like there's a lot of "noise" in the files and we could have short comment at the top about "see xbeat.full.yml for a commented example with all options" and then remove everything that not necessary. I like the readability of something like [this](https://github.com/elastic/beats-docker/blob/master/build/metricbeat/config/metricbeat.yml).